packages/x-charts/src/internals/plugins/corePlugins/useChartSeries/processSeries.ts

```diff
@@ -73,3 +74,37 @@ export const applySeriesProcessors = <TSeriesType extends ChartSeriesType>(
 
   return processedSeries;
 };
+
+/**
+ * Applies series processors with drawing area to series if defined.
+ * @param processedSeries The processed series groups
+ * @param seriesConfig The series configuration
+ * @param drawingArea The drawing area
+ * @returns Processed series with all transformations applied
+ */
+export const applySeriesLayout = <TSeriesType extends ChartSeriesType>(
+  processedSeries: ProcessedSeries<TSeriesType>,
+  seriesConfig: ChartSeriesConfig<TSeriesType>,
+  drawingArea: ChartDrawingArea,
+): SeriesLayout<TSeriesType> => {
+  let processingDetected = false;
+  const seriesLayout: SeriesLayout<TSeriesType> = {};
+
+  // Apply processors on series type per group
+  (Object.keys(processedSeries) as TSeriesType[]).forEach((type) => {
+    const processor = seriesConfig[type]?.seriesLayout;
+    if (processor !== undefined) {
+      const newValue = processor(processedSeries[type] as any, drawingArea);
+
+      if (newValue && newValue !== processedSeries[type]) {
+        processingDetected = true;
+        (seriesLayout as any)[type] = newValue;
+      }
+    }
+  });
```

Comment:
From my tests, we can remove `as any` by doing this:

```suggestion
  (Object.keys(processedSeries) as TSeriesType[]).forEach((type) => {
    const processor = seriesConfig[type]?.seriesLayout;
    const thisSeries = processedSeries[type];
    if (processor !== undefined && thisSeries !== undefined) {
      const newValue = processor(thisSeries, drawingArea);

      if (newValue && newValue !== processedSeries[type]) {
        processingDetected = true;
        seriesLayout[type] = newValue;
      }
    }
  });
```

Reply:
Thanks, That let me totice some other `as` was useless